<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CleaningRobotWindow</class>
 <widget class="QMainWindow" name="CleaningRobotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CleaningRobotWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="mainWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>933</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="RMainSetup">
     <attribute name="title">
      <string>main setup</string>
     </attribute>
     <widget class="QGroupBox" name="chooseRobotGroup">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>481</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>robot choosen</string>
      </property>
      <widget class="QCheckBox" name="rcr1CheckBox">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>34</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>1_left_UP</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="rcr2CheckBox">
       <property name="geometry">
        <rect>
         <x>137</x>
         <y>34</y>
         <width>85</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>2_Right_UP</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="rcr3CheckBox">
       <property name="geometry">
        <rect>
         <x>254</x>
         <y>34</y>
         <width>96</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>3_left_DOWN</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="rcr4CheckBox">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>34</y>
         <width>106</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>4_Right_DOWN</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>91</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>direction</string>
      </property>
      <widget class="QPushButton" name="downButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>DOWN</string>
       </property>
      </widget>
      <widget class="QPushButton" name="upButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>UP</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="robotEditframe">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>90</y>
        <width>381</width>
        <height>231</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QGroupBox" name="rcr1GroupBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>1_left_UP</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>150</width>
          <height>27</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="height">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rcr1HeightEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>150</width>
          <height>27</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="speed_2">
           <property name="text">
            <string>Speed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rcr1SpeedEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="rcr2GroupBox">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>10</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>2_Right_UP</string>
       </property>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>150</width>
          <height>27</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rcr2HeightEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>150</width>
          <height>27</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Speed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rcr2SpeedEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="rcr3GroupBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>3_left_DOWN</string>
       </property>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>150</width>
          <height>27</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rcr3HeightEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>150</width>
          <height>27</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Speed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rcr3SpeedEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="rcr4GroupBox">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>120</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>4_Right_DOWN</string>
       </property>
       <widget class="QWidget" name="layoutWidget_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>150</width>
          <height>27</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rcr4HeightEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>150</width>
          <height>27</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Speed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rcr4SpeedEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QTextEdit" name="RStatusEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>641</width>
        <height>221</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QGroupBox" name="serialPortGroup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>Serial Setup</string>
      </property>
      <widget class="QComboBox" name="cmbPortName">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>72</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbParity">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>160</y>
         <width>72</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="parity">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Parity</string>
       </property>
      </widget>
      <widget class="QLabel" name="bits">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Bits</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbStopBits">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>200</y>
         <width>72</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbFlowControl">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>240</y>
         <width>72</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbBaudRate">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>72</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Serial</string>
       </property>
      </widget>
      <widget class="QLabel" name="baud">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Baud</string>
       </property>
      </widget>
      <widget class="QLabel" name="stop">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
      <widget class="QLabel" name="flow">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Flow</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbDataBits">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>120</y>
         <width>72</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="serialOpenButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
      <widget class="QPushButton" name="serialSetButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>280</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>330</y>
        <width>451</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QPushButton" name="StopButton">
         <property name="text">
          <string>STOP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetButton">
         <property name="text">
          <string>RESET</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>CLEAR</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="RDataPage">
     <attribute name="title">
      <string>detail data</string>
     </attribute>
     <widget class="QGroupBox" name="rcr1DataBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>271</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>RCR 1</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>111</width>
         <height>217</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="speed">
          <property name="text">
           <string>Speed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="RCR1Speed_detaildata"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>111</width>
         <height>217</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="height_5">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="RCR1Height_detaildata"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="rcr2DataBox">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>271</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>RCR 2</string>
      </property>
      <widget class="QWidget" name="layoutWidget_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>111</width>
         <height>217</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="speed_3">
          <property name="text">
           <string>Speed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="RCR2Speed_detaildata"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_9">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>111</width>
         <height>217</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="height_2">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="RCR2Height_detaildata"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="rcr2DataBox_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>320</y>
        <width>271</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>RCR 4</string>
      </property>
      <widget class="QWidget" name="layoutWidget_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>111</width>
         <height>217</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="speed_4">
          <property name="text">
           <string>Speed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="RCR4Speed_detaildata"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_11">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>111</width>
         <height>217</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="height_3">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="RCR4Height_detaildata"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="rcr1DataBox_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>320</y>
        <width>271</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>RCR 3</string>
      </property>
      <widget class="QWidget" name="layoutWidget_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>111</width>
         <height>217</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="speed_5">
          <property name="text">
           <string>Speed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="RCR3Speed_detaildata"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_13">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>111</width>
         <height>217</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="height_4">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="RCR3Height_detaildata"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="clearRCR1Button">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>300</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear RCR1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearRCR2Button">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>300</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear RCR2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearRCR4Button">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>610</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear RCR4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearRCR3Button">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>610</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear RCR3</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="RSingleSetup">
     <attribute name="title">
      <string>single setup</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain_setup">
    <property name="title">
     <string>main setup</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSingle_setup">
    <property name="title">
     <string>single setup</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAdd_plot">
    <property name="title">
     <string>add plot</string>
    </property>
   </widget>
   <addaction name="menuMain_setup"/>
   <addaction name="menuSingle_setup"/>
   <addaction name="menuAdd_plot"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
